compiler/resolution/functionResolution.cpp

```diff
@@ -4124,6 +4124,26 @@ void printResolutionErrorAmbiguous(CallInfo&                  info,
   USR_STOP();
 }
 
+static bool isMethodPreResolve(CallExpr* call) {
+  bool result = false;
+
+  if (call->numActuals() >= 2)
```

Comment:
Should this be `>= 1`?

Reply:
Old code only considered it a method if there were at least two actuals and the first is the method token, not sure if that's correct, but that's where I got the code from.

```cpp
 bool isMethod = false;
  if (call->numActuals() >= 2) {	
    if (SymExpr* se = toSymExpr(call->get(1))) {	
      isMethod = se->symbol() == gMethodToken;	
    }	
  }
```